cli/src/complete.rs

```diff
@@ -28,19 +29,42 @@ use crate::config::default_config;
 use crate::config::LayeredConfigs;
 use crate::ui::Ui;
 
+static BOOKMARK_HELP_TMPL: &str = r#"
+"\t" ++
+if(normal_target,
+    normal_target.change_id().shortest() ++ ": " ++
+    if(normal_target.description(),
+        normal_target.description().first_line(),
+        "(no description set)",
+    ),
+    "(conflicted bookmark)",
+)"#;
+
+/// A helper function for various bookmark completer functions. It returns
+/// (name, help) from a line formatted with `BOOKMARK_HELP_TMPL`.
+fn bookmark_line_to_name_and_help(line: &str) -> (&str, Option<StyledStr>) {
+    match line.split_once('\t') {
+        Some((name, help)) => (name, Some(help.to_string().into())),
+        None => (line, None),
+    }
```

Comment:
nit: this could be a generic `split_help_text()` function. I think we have a similar pattern in op id completion.